### DISCUSSION TITLE
请加个时钟　Add a clock please, like this...

### DISCUSSION THREAD

**Discussion by xyn19** [zhongyang219/TrafficMonitor#1282] [Ideas]
![image](https://user-images.githubusercontent.com/64721650/185459217-36994f44-2c91-4157-acb5-1cc175b2cc5d.png)

Shows seconds 秒

So I can ditch the CPU hungry ElevenClock, thank you!

**Comment by zhongyang219** [comment]
The [date time plugin](https://github.com/zhongyang219/TrafficMonitorPlugins/blob/main/download/plugin_download_en.md) may fit your needs.

**Reply by williwxie** [reply]
> The [date time plugin](https://github.com/zhongyang219/TrafficMonitorPlugins/blob/main/download/plugin_download_en.md) may fit your needs.

显示倒是可以，但是这个貌似并不能在原来的位置显示，而是又显示个新的。问题解决了，但没有完全解决。
![image](https://user-images.githubusercontent.com/46312111/209446748-6bf07722-1cd2-408a-b683-50afb9a99083.png)
